react-accordion/my-app/src/TopicCard.tsx

```diff
@@ -0,0 +1,20 @@
+type Props = {
+  topics: { id: number; title: string; content: string }[];
+  handleClick: (selectedTopic: number) => void;
```

Comment:
Your prop names should start with "on". The function names start with "handle", but not the prop names.